lib/styles/components/_buttons.scss

```diff
@@ -291,21 +291,31 @@ $button-disabled-alpha: 0.6;
 		}
 	}
 
-	&.-elevation-x-small {
+	&.-elevation-x-small,
+	&.-elevation-small {
 		&.-outlined {
-			&:not(.-hovered):not(:hover):not(.-focused):not(:focus):not(.-disabled) {
-				background: $color-default-background !important;
+			&::before {
+				background: $color-default-background;
+				border-radius: 100px;
+				bottom: 0;
+				content: '';
+				display: block;
+				left: 0;
+				position: absolute;
+				right: 0;
+				top: 0;
+				transform: translateZ(-1px);
 			}
 		}
-	}
 
-	&.-elevation-small {
-		&.-outlined {
-			&:not(.-hovered):not(:hover):not(.-focused):not(:focus):not(.-disabled) {
-				background: $color-default-background !important;
+		&.-rounded {
```

Comment:
we use `::before` only for `-outlined`. For safety, I'd move this block inside `&.-outlined {` block

Reply:
@kvas-damian i fixed in in the wrong branch: https://github.com/bethinkpl/design-system/blob/b5fb1bfd3cc51cb25f902a949aa68a48751b44de/lib/styles/components/_buttons.scss#L220